src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -95,21 +94,17 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
                     if (!Path.IsPathRooted(assemblyName))
                         assemblyName = Path.Combine(Environment.CurrentDirectory, assemblyName);
 
-                    TestLog.Info("Running all tests in " + assemblyName);
```

Comment:
As it happens, I agree with this change but, in general, please don't make unrelated changes when fixing a problem. If you feel that there are too many messages, that's really a separate issue and mixing different issues in one PR means it only gets merged if the reviewers agree with __all__ of them.